platform_buckets_api/kube_client.py

```diff
@@ -441,13 +291,14 @@ async def list_user_buckets(
             label_selectors.append(f"{OWNER_LABEL}={owner}")
         if name:
             label_selectors.append(f"{BUCKET_NAME_LABEL}={name}")
-        if org_name and org_name.upper() == NO_ORG:
-            label_selectors.append(f"!{ORG_NAME_LABEL}")
-        elif org_name:
-            label_selectors.append(f"{ORG_NAME_LABEL}={org_name}")
+        if org_name:
+            normalized_name = normalize_name(org_name)
+            if normalized_name != normalize_name(NO_ORG):
+                normalized_name = org_name
+            label_selectors.append(f"{APOLO_ORG_NAME_LABEL}={normalized_name}")
         if label_selectors:
```

Comment:
Should we also add label selector for project_name? AFAIU after migration all buckets will have project label, right? Currently buckets may not have this label, that's why we retrieve all buckets and filter by project afterwards

Reply:
right! done